### DISCUSSION TITLE
“Eigen/Dense”: No such file or directory [E:\instant-ngp\build\optix_program.vcxproj]

### DISCUSSION THREAD

**Discussion by xiaoxinxin1314** [NVlabs/instant-ngp#295] [Q&A]
When I press "cmake --build build --config RelWithDebInfo -j 16",error occurred，like this，“Eigen/Dense”: No such file or directory [E:\instant-ngp\build\optix_program.vcxproj
![5](https://user-images.githubusercontent.com/90880551/155315101-7457dd1d-1cce-483b-b29e-f2021a27a608.png)
could you give me some suggestions, thanks

**Comment by Tom94** [comment]
You're likely missing some of the submodules.

```sh
git submodule init
git submodule update --recursive
```

**Reply by zywzywzyw** [reply]
I tried your solution but it doesn't seem to work

**Reply by suwcg** [reply]
I come across the same question, do you solve it?
